### PR TITLE
Replace zvm autoyast profile with patched profile

### PR DESCRIPTION
**Pull request by ge0r**
Generating an autoyast profile with the fix for [bsc#1196963](https://bugzilla.suse.com/show_bug.cgi?id=1196963) resolves the [autoyast_zvm failures](https://openqa.suse.de/tests/8344918#).

- Related ticket: No ticket
- Needles: No needles
- Verification run: https://openqa.suse.de/tests/8352578#

### CODE REVIEW

File: data/autoyast_sle15/autoyast_sles_zvm.xml [hunk 1]
```diff
@@ -1,1006 +1,1395 @@
...
+    <group t="map">
+      <gid>479</gid>
+      <groupname>pulse</groupname>
+      <userlist/>
+    </group>
+  </groups>
+  <host t="map">
+    <hosts t="list">
+      <hosts_entry t="map">
+        <host_address>10.162.42.101</host_address>
```
Comment (jknphy) [inline data/autoyast_sle15/autoyast_sles_zvm.xml:475]: this `host_entry` is pointing to specific IP, I guess it is better to remove it.
or perhaps replace it with `{{HostIP}}` but the original didn't need it ?

File: data/autoyast_sle15/autoyast_sles_zvm.xml [hunk 2]
```diff
@@ -1,1006 +1,1395 @@
...
+    </general>
+  </kdump>
+  <networking t="map">
+    <dhcp_options t="map">
+      <dhclient_client_id/>
+      <dhclient_hostname_option>AUTO</dhclient_hostname_option>
+    </dhcp_options>
+    <dns t="map">
+      <dhcp_hostname t="boolean">true</dhcp_hostname>
+      <hostname>s390qa101.qa.suse.de</hostname>
```
Comment (jknphy) [inline data/autoyast_sle15/autoyast_sles_zvm.xml:565]: this reference is also specific, we should remove it IMO.

File: data/autoyast_sle15/autoyast_sles_zvm.xml [hunk 3]
```diff
@@ -1,1006 +1,1395 @@
...
+        <channel>0.0.0200</channel>
+        <diag t="boolean">false</diag>
+        <format t="boolean">false</format>
+      </listentry>
+      <listentry t="map">
+        <channel>0.0.ffff</channel>
+        <diag t="boolean">false</diag>
+        <format t="boolean">false</format>
+      </listentry>
+      <listentry t="map">
```
Comment (jknphy) [inline data/autoyast_sle15/autoyast_sles_zvm.xml:57]: we could delete all listentries except this `0.0.0150`.
The shorter the file the better, but it is quite tricky in this exotic archs. if you see something else to remove that is fine.

File: data/autoyast_sle15/autoyast_sles_zvm.xml (hunk 1 above)
Comment (ge0r) [reply data/autoyast_sle15/autoyast_sles_zvm.xml:475]: Wow I forgot about this, thanks!

File: data/autoyast_sle15/autoyast_sles_zvm.xml (hunk 2 above)
Comment (ge0r) [reply data/autoyast_sle15/autoyast_sles_zvm.xml:565]: same here yes

**Pull request merged**